src/style.css

```diff
@@ -1,3 +1,76 @@
 body {
+  --hover-link-color: #00acdc;
   margin: 0;
 }
+
+.header {
+  box-sizing: border-box;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  position: absolute;
+  width: 100%;
+  padding-right: 50px;
+  padding-left: 50px;
+}
+
+.header-nav {
+  align-items: center;
+  display: flex;
+}
+
+.nav-list {
+  padding: 0;
+  margin: 0;
+  display: flex;
+  list-style-type: none;
+}
+
+.nav-list-item {
+  display: flex;
+  align-items: center;
+}
+
+.nav-link {
+  display: inline-block;
+  text-align: center;
+  text-wrap: nowrap;
+  text-transform: uppercase;
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+  font-style: normal;
+  font-weight: 500;
+  line-height: 60px;
+  text-decoration: none;
+  height: 60px;
+  margin-left: 20px;
+  color: #000;
+}
+
+.nav-link:hover {
+  color: var(--hover-link-color);
+}
+
+.nav-list :first-child > .nav-link {
```

Comment:
[Increasing an Element Specificity](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules#typical-bem-mistakes-in-css)